allen-cell-animated/cell-feature-data
# Feature/GitHub action publish to staging

Problem
=======
What is the problem this work solves, including
closes #49 

Solution
========
What I/we did to solve this problem
- Added repo secrets in github for db access 
- Created a yaml file in github/workflows that will be manually triggered and run process-dataset 
- Input includes the path to the dataset and a boolean that indicates whether to skip the file info upload 


with @meganrm 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update (we can update the documentation when the release dataset to production db is done)


Steps to Verify:
----------------
1. Go to Actions and choose `upload dataset to staging DB` in workflows 
2. Click on `Run Workflow` and select `feature/github-action-publish-to-staging` branch  
3. Then enter the path to the new dataset
4. Check the box if we want to skip the file-info upload 
5. Run workflow
